conclave-common/src/main/kotlin/com/r3/conclave/common/internal/attestation/AttestationJsonTypes.kt

```diff
@@ -332,46 +318,18 @@ data class Tcb constructor(
         }
 
         private fun fromJsonV2(json: JsonNode): Tcb {
-            return Tcb(
-                    json.getInt("sgxtcbcomp01svn"),
-                    json.getInt("sgxtcbcomp02svn"),
-                    json.getInt("sgxtcbcomp03svn"),
-                    json.getInt("sgxtcbcomp04svn"),
-                    json.getInt("sgxtcbcomp05svn"),
-                    json.getInt("sgxtcbcomp06svn"),
-                    json.getInt("sgxtcbcomp07svn"),
-                    json.getInt("sgxtcbcomp08svn"),
-                    json.getInt("sgxtcbcomp09svn"),
-                    json.getInt("sgxtcbcomp10svn"),
-                    json.getInt("sgxtcbcomp11svn"),
-                    json.getInt("sgxtcbcomp12svn"),
-                    json.getInt("sgxtcbcomp13svn"),
-                    json.getInt("sgxtcbcomp14svn"),
-                    json.getInt("sgxtcbcomp15svn"),
-                    json.getInt("sgxtcbcomp16svn"),
-                    json.getInt("pcesvn")
-            )
+            val sgxtcbcompsvn = ArrayList<Int>()
+            for (i in 1..16) {
+                val elementString = i.toString().padStart(2, '0')
+                sgxtcbcompsvn.add(json.getInt("sgxtcbcomp${elementString}svn"))
+            }
+            return Tcb(sgxtcbcompsvn, json.getInt("pcesvn"))
```

Comment:
Can you wrap `sgxtcbcompsvn` inside `Collections.unmodifiableList`. Just being super careful in making sure these data structures are immutable.